data/crac-creation/crac-creator-csa-profiles/src/test/resources/parameters/csa-crac-parameters-json-csa-crac-creation-parameters-test.json

```diff
@@ -0,0 +1,9 @@
+{
+  "crac-factory" : "CracImplFactory",
+  "extensions" : {
+    "CsaCracCreatorParameters" : {
+      "capacity-calculation-region-eic-code": "10Y1001C--00095L-1",
```

Comment:
Can you use 10Y1001C--00095L EIC instead please? (better to use a real EI code)

Reply:
I change it to RTE EI code to have a real EI code but different from default EI code